packaging/test-install.sh

```
@@ -0,0 +1,233 @@
+  up=
+  for _ in $(seq 1 60); do
+    docker exec "app-$$" curl -fsS http://127.0.0.1:8081/up >/dev/null 2>&1 && { up=1; break; }
+    sleep 1
+  done
+
+  if [ -z "$up" ]; then
+    bad "configured server never answered /up"
+  else
+    ok "starts and applies its migrations"
+    grep -q 'true' <<<"$(docker exec "app-$$" curl -s http://127.0.0.1:8081/setup/status)" \
+      && ok "the first-run wizard is offered" || bad "setup_required was not true"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)